qemu/tests/transfer_file_over_ipv6.py

```diff
@@ -72,14 +75,28 @@ def get_file_md5sum(file_name, session, timeout):
 
     # config ipv6 address host and guest.
     host_ifname = params.get("netdst")
+    host_address = utils_net.get_host_ip_address(
+        params, ip_ver="ipv6", linklocal=True)
 
+    error_context.context("Get ipv6 address of host: %s" % host_address,
+                          logging.info)
     for vm in vms:
         vm.verify_alive()
         sessions[vm] = vm.wait_for_login(timeout=timeout)
-        inet_name[vm] = utils_net.get_linux_ifname(sessions[vm],
-                                                   vm.get_mac_address())
-        addresses[vm] = get_linux_ipv6_linklocal_address(inet_name[vm],
-                                                         sessions[vm])
+        if params.get("os_type") == "linux":
+            inet_name[vm] = utils_net.get_linux_ifname(
+                sessions[vm], vm.get_mac_address())
+            addresses[vm] = utils_net.get_guest_ip_addr(
+                sessions[vm],
+                vm.get_mac_address(),
+                params.get("os_type"),
+                ip_version="ipv6",
+                linklocal=True)
+        else:
+            addresses[vm] = get_windows_ipv6_linklocal_address(sessions[vm])
```

Comment:
any reason not to use utils_net.get_guest_ip_addr to get guest ip in win guest?

Reply:
@xiywang  Changed avocado-vt to support windows guest for ipv6 locallink. So already updated to use utils_net.get_guest_ip_addr for win guest.